drivers/clock_control/clock_stm32g0.c

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Look very similar to `clock_stm32l4_wb.c`. Is it possible to use `clock_stm32l4_wb.c` instead of `clock_stm32g0.c`?

Reply:
@FRASTM From your comment below, it is possible to combine Kconfig.stm32l4_wb and Kconfig.stm32g0 in a  Kconfig.stm32l4_wb_g0. Could we do the same for clock_stm32l4_wb.c and clock_stm32g0.c?